OOVPA.h

```diff
@@ -182,14 +209,18 @@ typedef struct _OOVPATable
      _9,  _8,  _7,  _6,  _5,  _4,  _3,  _2,  _1,  _0, \
     N, ...) N
 
-// NOTE: A reminder for all of developers, TYPE argument is meant for research documentation.
-// (eventually UNPATCHED will be replace back to PATCH once LLE is fully supportive in the future)
 #define REGISTER_OOVPAS(Symbol, ...) MSVC_EXPAND(_GET_NTH_ARG("ignored", __VA_ARGS__, \
     REGISTER_OOVPA_19, REGISTER_OOVPA_18, REGISTER_OOVPA_17, REGISTER_OOVPA_16, REGISTER_OOVPA_15, \
     REGISTER_OOVPA_14, REGISTER_OOVPA_13, REGISTER_OOVPA_12, REGISTER_OOVPA_11, REGISTER_OOVPA_10, \
     REGISTER_OOVPA_9, REGISTER_OOVPA_8, REGISTER_OOVPA_7, REGISTER_OOVPA_6, REGISTER_OOVPA_5, \
     REGISTER_OOVPA_4, REGISTER_OOVPA_3, REGISTER_OOVPA_2, REGISTER_OOVPA_1, REGISTER_OOVPA_0)(Symbol, __VA_ARGS__))
 
+#define OV_ENTRIES(Offset, ...) MSVC_EXPAND(_GET_NTH_ARG("ignored", __VA_ARGS__, \
```

Comment:
nit: I think a name like `OV_BYTES` would be more fitting. As the macro is only used in those files, you could choose an even shorter or descriptive name without prefix (my original proposal on XboxDev Discord was `MATCH(0x####, 0x##, 0x##, ...)`).

I also think the trailing comma could be part of the macro, so that it will eventually look like a domain specific language, parsable by trivial non-C compilers.

This can also be seen in my original proposal:

```
        // D3DDevice_GetBackBuffer+0x04 : cmp eax, 0xFFFFFFFF
        MATCH(0x0004,     0x83, 0xF8, 0xFF)

        // D3DDevice_GetBackBuffer+0x0D : jnz +0x07
        MATCH(0x000D,     0x75, 0x07)
```

Reply:
Been replaced to OV_MATCH